src/scanners/scan-functions/aws/redshift-clusters.ts

```diff
@@ -0,0 +1,41 @@
+import {RedshiftClient, DescribeClustersCommand, Cluster} from '@aws-sdk/client-redshift'
+import {RateLimiter} from '@/scanners/common/RateLimiter'
+import {buildARN} from './common/buildArn'
+import {Credentials, Resources} from '@/types'
+import {getAwsAccountId} from './common/getAwsAccountId'
+
+export async function getRedshiftClusters(
+	credentials: Credentials,
+	rateLimiter: RateLimiter,
+	region: string,
+): Promise<Resources<Cluster>> {
```

Comment:
Looking at the `Cluster` type, it contains `VpcId` and `AvailabilityZone`, which we also need to add to the processed data in `src/aws-app/process/getProcessedResource.ts`. Can you please take care of those as well?

Reply:
Thanks for the catch!